app/utils/selectStarterTemplate.ts

```diff
@@ -179,8 +179,10 @@ const getGitHubRepoContent = async (repoName: string, path: string = '', env?: E
       headers.Authorization = 'token ' + token;
     }
 
+    const ref = env?.USE_PRODUCTION_TEMPLATE === 'true' ? 'production' : 'main';
```

Comment:
IMHO, `production` / `main` seems little bit ambigious. how about `production` / `development`?

Reply:
I agree. Since our developers are working hard on that `main` branch, I just added `production` instead of seprating to main / develop.